<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QClassRemindWindow</class>
 <widget class="C8CommonWindow" name="QClassRemindWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QClassRemindWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_classRemind" native="true">
     <property name="minimumSize">
      <size>
       <width>393</width>
       <height>285</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_titleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_remding_title">
           <property name="text">
            <string>RemdingTitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_IsClassOver">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>393</width>
            <height>230</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>22</number>
               </property>
               <property name="leftMargin">
                <number>52</number>
               </property>
               <property name="topMargin">
                <number>81</number>
               </property>
               <property name="rightMargin">
                <number>52</number>
               </property>
               <item>
                <widget class="QLabel" name="label_class_remind">
                 <property name="minimumSize">
                  <size>
                   <width>37</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>37</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_isclassover_info">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>IsClassOver?</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>112</number>
               </property>
               <property name="topMargin">
                <number>85</number>
               </property>
               <property name="rightMargin">
                <number>111</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_OK">
                 <property name="minimumSize">
                  <size>
                   <width>74</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>OK</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_Cancel">
                 <property name="minimumSize">
                  <size>
                   <width>74</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Cancel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_ClassOver">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>14</number>
               </property>
               <property name="leftMargin">
                <number>36</number>
               </property>
               <property name="topMargin">
                <number>44</number>
               </property>
               <property name="rightMargin">
                <number>36</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>37</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>37</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>148</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_class_remind_2">
                    <property name="minimumSize">
                     <size>
                      <width>37</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>37</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="topMargin">
                  <number>13</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_classover_remind">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>TheClassOver</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>50</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_plan_time">
                     <property name="text">
                      <string>PlanTime</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_actual_time">
                     <property name="text">
                      <string>ActualTime</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_latency_time">
                     <property name="text">
                      <string>LatencyTime</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <property name="topMargin">
                      <number>10</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="pushButton_iknow">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>74</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>74</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="layoutDirection">
                        <enum>Qt::LeftToRight</enum>
                       </property>
                       <property name="text">
                        <string>IKnow</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_is_plan_classover">
         <widget class="QWidget" name="widget_5" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>393</width>
            <height>230</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <property name="spacing">
                <number>14</number>
               </property>
               <property name="leftMargin">
                <number>52</number>
               </property>
               <property name="topMargin">
                <number>81</number>
               </property>
               <property name="rightMargin">
                <number>52</number>
               </property>
               <item>
                <widget class="QLabel" name="label_class_remind_3">
                 <property name="minimumSize">
                  <size>
                   <width>37</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>37</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_planclassover">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>IsPlanClassOver</string>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>84</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <property name="leftMargin">
                      <number>18</number>
                     </property>
                     <property name="topMargin">
                      <number>8</number>
                     </property>
                     <property name="rightMargin">
                      <number>41</number>
                     </property>
                     <property name="bottomMargin">
                      <number>27</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_plan_remind">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>20</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Close=PlanClassOver</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_13">
                     <property name="spacing">
                      <number>13</number>
                     </property>
                     <property name="topMargin">
                      <number>35</number>
                     </property>
                     <property name="rightMargin">
                      <number>84</number>
                     </property>
                     <property name="bottomMargin">
                      <number>1</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="pushButton_retard_classover">
                       <property name="minimumSize">
                        <size>
                         <width>74</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Retard10Min</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_plan_classover">
                       <property name="minimumSize">
                        <size>
                         <width>74</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>PlannedOverClass</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_wait_start_class">
         <widget class="QWidget" name="widget_6" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>393</width>
            <height>230</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <property name="spacing">
                <number>22</number>
               </property>
               <property name="leftMargin">
                <number>52</number>
               </property>
               <property name="topMargin">
                <number>80</number>
               </property>
               <property name="rightMargin">
                <number>52</number>
               </property>
               <item>
                <widget class="QLabel" name="label_class_remind_4">
                 <property name="minimumSize">
                  <size>
                   <width>37</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>37</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_wait_startclass">
                 <property name="text">
                  <string>WaitStartClass</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>157</number>
               </property>
               <property name="topMargin">
                <number>90</number>
               </property>
               <property name="rightMargin">
                <number>156</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_iknow_2">
                 <property name="minimumSize">
                  <size>
                   <width>74</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>IKnow</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>C8CommonWindow</class>
   <extends>QWidget</extends>
   <header>c8commonwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
